container/apptabs.go

```diff
@@ -60,6 +66,15 @@ func (t *AppTabs) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
+// Append adds a new TabItem to the end of the tab bar.
+//
+// Deprecated: Moved into an embedded type.
```

Comment:
These don't need to be deprecated as far as I can tell, moving a function to an embedded type in the future is a transparent operation.

Reply:
Yeah I wasn't sure how to articulate this, it should not affect any devs, but at the same time I want to tag this method for removal. Maybe `//TODO @stuartmscott Remove in next major`?

Reply:
Moot